merge-structure-cli/src/test/java/community/redrover/merge/cli/CliUtilsTest.java

```diff
@@ -0,0 +1,163 @@
+package community.redrover.merge.cli;
+
+import com.beust.jcommander.JCommander;
+import community.redrover.merge.model.Strategy;
+import community.redrover.merge.model.config.AppendStrategyConfig;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.*;
+import community.redrover.merge.testutils.TempFile;
+
+public class CliUtilsTest {
+
+    @Test
+    void testIsInvalidPathNull() {
+        assertTrue(CliUtils.isInvalidPath(null));
+    }
+
+    @Test
+    void testIsInvalidPathBlank() {
+        assertTrue(CliUtils.isInvalidPath("   "));
+    }
+
+    @Test
+    void testIsInvalidPathStartsWithDash() {
+        assertTrue(CliUtils.isInvalidPath("--source"));
+    }
+
+    @Test
+    void testIsInvalidPathValid() {
+        assertFalse(CliUtils.isInvalidPath("valid/path.yaml"));
+    }
+
+    @Test
+    void testParseArgsHelpFlagThrowsCliException() {
+        String[] args = {"--help"};
+
+        CliException ex = assertThrows(CliException.class, () -> CliUtils.parseArgs("append", args));
+
+        assertTrue(ex.shouldShowUsage());
+        assertNull(ex.getMessage());
+    }
+
+    @Test
+    void testParseArgsInvalidArgsThrowsCliException() {
+        String[] args = {"--unknown"};
+
+        CliException ex = assertThrows(CliException.class, () -> CliUtils.parseArgs("append", args));
+
+        assertTrue(ex.shouldShowUsage());
+        assertTrue(ex.getMessage().startsWith("Error:"));
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = { "json", "yaml" })
```

Comment:
adding "yml" as we allow it may help to test all cases
maybe use something like following to get array of all supported extentions:
`Arrays.stream(SupportedExtension.values()).map(Enum::name).toArray(String[]::new)`